src/main/java/org/opensearch/security/rest/SecurityInfoAction.java

```diff
@@ -73,6 +73,8 @@
     private final PrivilegesEvaluator evaluator;
     private final ThreadContext threadContext;
 
+    public static final String AUTHINFO_SUFFIX = "authinfo";
```

Comment:
The same as API_AUTHTOKEN_SUFFIX. Additionally static import should be separated from not static imports.